code/src/sixsq/nuvla/server/resources/infrastructure_service/utils.clj

```diff
@@ -0,0 +1,21 @@
+(ns sixsq.nuvla.server.resources.infrastructure-service.utils
+  (:require
+    [sixsq.nuvla.server.resources.common.schema :as c]
+    [sixsq.nuvla.server.util.response :as r]))
+
+
+(defn can-delete?
```

Comment:
This function should return a boolean or have a name that doesn't end with '?'.

Reply:
simplified